packages/mainmenu-extension/src/index.ts

```diff
@@ -342,13 +347,13 @@ export function createFileMenu(
     execute: Private.delegateExecute(app, menu.consoleCreators, 'createConsole')
   });
 
-  commands.addCommand(CommandIDs.quit, {
-    label: 'Quit',
-    caption: 'Quit JupyterLab',
+  commands.addCommand(CommandIDs.shutdown, {
+    label: 'Shutdown',
+    caption: 'Shut down JupyterLab',
     execute: () => {
       return showDialog({
-        title: 'Quit confirmation',
-        body: 'Please confirm you want to quit JupyterLab.',
+        title: 'Shut down confirmation',
+        body: 'Please confirm you want to shut down JupyterLab.',
         buttons: [Dialog.cancelButton(), Dialog.warnButton({ label: 'Quit' })]
```

Comment:
This warn button should probably be changed to "Shut Down"

Reply:
I think @jasongrout preferred "Shut down" :)

Reply:
I'm referring to the button label,  which is still "quit", as far as I can tell

Reply:
But since the buttons are transformed to upper case, it shouldn't really matter.

Reply:
(For reference on language usage, see http://notaverb.com/shutdown).

I think @ian-r-rose is saying that this last line should also be changed:
```typescript
buttons: [Dialog.cancelButton(), Dialog.warnButton({ label: 'Quit' })]
```
to
```typescript
buttons: [Dialog.cancelButton(), Dialog.warnButton({ label: 'Shut Down' })]
```

Reply:
Oh, I didn't notice. Fixed this. Sorry to drag this along :-/